tests/test_equilibrium.py

```diff
@@ -369,3 +374,50 @@ def test_backward_compatible_load_and_resolve():
     f_obj = ForceBalance(eq=eq)
     obj = ObjectiveFunction(f_obj, use_jit=False)
     eq.solve(maxiter=1, objective=obj)
+
+
+@pytest.mark.unit
+def test_contract_equilibrium():
```

Comment:
This only tests that the values at the new boundary match the values at the old internal surface. Suggestion for a new test: do a plotting image test with `plot_comparison((eq, eq_half_rho))` to check that the flux surfaces actually match on all of the surfaces

Reply:
It also tests if |B| and |F| are the same, which are functions of the surface geometry. I can add R Z and lambda as well to the tests

Reply:
My point is that we should also test at some intermediate surface, not just the boundary. If you add another couple of surfaces to the comparison in this test I'll be happy.

Reply:
testing `data_keys = ["|B|", "|F|", "R", "Z", "lambda", "p", "iota"]` in the volume of the smaller equilibrium is literally testing at multiple surfaces, so this test covers what you want.